src/LibChorus/Model/ServerSettingsModel.cs

```diff
@@ -352,7 +353,15 @@ internal static string EncryptPassword(string encryptMe)
 			{
 				return encryptMe;
 			}
-			var encryptedData = ProtectedData.Protect(Encoding.Unicode.GetBytes(encryptMe), Encoding.Unicode.GetBytes(EntropyValue), DataProtectionScope.CurrentUser);
+
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) &&
+				!RuntimeInformation.FrameworkDescription.Contains("Framework"))
```

Comment:
This doesn't work when testing on my local machine. When running with Mono `RuntimeInformation.FrameworkDescription` returns `Mono 6.12.0.182 (tarball Tue Jun 14 22:35:00 UTC 2022)`.

Reply:
Ok, I'm unable to see this on my Linux box. Should I replace Framework with something? Maybe Mono? Do I need another condition? Or I could try deleting it?

Reply:
What command are you using to run with Mono? It's quite possible I'm not using the right thing

Reply:
I reverted this code. It's unrelated to the core bug, which was just Platform.IsMono -> .IsLinux, and created [a separate issue for it](https://github.com/sillsdev/chorus/issues/298).

Reply:
I was stepping through the net461 tests in the debugger in Rider on Ubuntu 22.04.

Reply:
Ok. Yeah I have less of a pure Linux environment, being on WSL, so I trust your PC more than mine